eos/parameters/sm-and-eft.yaml

```diff
@@ -799,124 +817,154 @@ groups:
           central: 1.0
           min:     1.0
           max:     1.0
+          latex:   '$\operatorname*{Re} \mathcal{C}^{b\to u \ell^-\bar\nu}_{V_L}$'
```

Comment:
Why don't we just use `\Re` here?

Reply:
Because ``\Re`` uses some hardly legible character, rather than simply ``Re`` in the roman upright font.

Reply:
I see :)